docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,86 @@
+
+Job opportunities
+-----------------
+
+In recent years we noticed that an increasing number of documentarians in our communities are looking to evolve their careers, and that many employers are seeking documentation-friendly folks to add to their organizations.
+Aside from the traditional sponsorship options for the conferences, we have started two very exciting experiments:
+
+Job board
+~~~~~~~~~
+
+As we mentioned in previous newsletters and Slack posts, we are launching a new job board, led by Kelly O'Brien, which aims to make it easy for documentarians to find their dream gigs, and for employers to find their dream documentarians.
+
+We're currently in the beta-testing phase, so if you're interested in helping us test the board – either as an employer or an applicant – we'd love to hear from you at `jobs@writethedocs.org <mailto:jobs@writethedocs.org>`_.
+
+Job fair at the conference
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The 2018 edition of our sponsorship offering for both the Portland and Prague conference now includes an additional **Job Fair** package that focuses on connecting potential employers with job seekers.
+During the morning of the second conference day (Tuesday), the unconference room will be transformed into a job fair where tables are assigned to organizations who are looking to hire documentarians.
+
+Conference attendees will be able to head over to the job fair to find out more information about job opportunities in a comfortable and friendly setting, and companies will be able to optimize their hiring efforts into these time slots.
+Space in the job fair is limited by the room capacity, so if you want your company to participate, write to us at sponsorship@writethedocs.org to reserve your slot.
+
+Swag
+----
+
+Some of you might have noticed that this year's conference tickets do not include shirts.
+We realized that after 5 years of printing shirts for multiple conferences, the swag offering could use a bit of a refresh.
+In addition, the overhead of printing hundreds of shirts and managing inventory simply proved ineffective over time.
+
+So this year we are changing swag tactics, and instead of including shirts with the ticket purchase, we happily present **tote bags**! No sizing required, smoother production workflow, and just as collectible :)
+
+But have no fear, you can still get a shirt for your favorite conference! Over the next few weeks we will be setting up a print-on-demand campaign where you can order a shirt with this year's conference design, and anyone who orders by a certain date should receive their shirt in time for the conference.
+
+We will continue to print speaker and staff hoodies, since we can estimate sizing and quanitities much more accurately than we can with shirts. If you want to help organize one of our conferences, write to us at support@writethedocs.org and join the fun!
```

Comment:
Maaaaaybe remove this line? Not too fussed. But sneaky nice tie in with help us organis.